types/ember__string/index.d.ts

```diff
@@ -1,18 +1,23 @@
 // Type definitions for non-npm package @ember/string 3.0
 // Project: https://emberjs.com/api/ember/3.4/modules/@ember%2Fstring
 // Definitions by: Mike North <https://github.com/mike-north>
+//                 Scott Kidder <https://github.com/scottkidder>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
-import { SafeString } from "./-private/handlebars";
+import { SafeString } from "@ember/template/-private/handlebars";
+
+/** @deprecated use import from '@ember/template' instead. */
+export function htmlSafe(str: string): SafeString;
+
+/** @deprecated use import from '@ember/template' instead. */
+export function isHTMLSafe(str: any): str is SafeString;
```

Comment:
Instead of redefining these here, IMHO it's better to re-export them from `@ember/template`:
```ts
export { htmlSafe, isHTMLSafe } from '@ember/template';
```
That way we keep a single definition for these functions and you don't have to import `SafeString` from `@ember/template/-private` here. I'm not sure if the deprecated annotation will be picked up by tslint doing it this way, but I'd still add it for documentation purposes.

Reply:
Yeah I've done that now. I agree, from what I could find I don't think `@deprecated` will picked up this way.